lib/config.js

```diff
@@ -11,7 +11,8 @@ let _config
 
 function readConfig () {
   return new BB((resolve, reject) => {
-    const child = spawn('npm', ['config', 'ls', '--json'], {
+    const npmBin = process.platform === 'win32' ? 'npm.cmd' : 'npm'
```

Comment:
Coincidentally, one alternative to this ternary is the package that caused the need for #8, [strong-spawn-npm](https://github.com/strongloop/strong-spawn-npm).